packaging/debian/config-default/changelog

```diff
@@ -1,3 +1,9 @@
+cvmfs-config-default (2.0-2) lucid; urgency=low
```

Comment:
Shouldn't that be 2.0-3?

Reply:
if it is synchronous with the non-debian package, yes. but the changelog here does not include the 2.0-2 which the other one has. not sure if forgotten or not?

Reply:
i tried to find the commit as the change log for the non-debian says 
```
* Sat Apr 10 2021 Jakob Blomer <jblomer@cern.ch> - 2.0-2
- Fix RPM linter errors
```

but i could not find it
```
commit 9660032838612c6f95ba201069175c02a34b0307
Author: Dave Dykstra <2129743+DrDaveD@users.noreply.github.com>
Date:   Mon Apr 12 09:14:43 2021 -0500

    add common SanitizeParameterAssignment function

commit 93179d74a6a631853d4042a3ace8dcc95db500cb
Author: Dave Dykstra <2129743+DrDaveD@users.noreply.github.com>
Date:   Thu Apr 8 17:02:38 2021 -0500

    skip attempting to parse parameters with spaces in them
```